sample-apps/delivery-customizations/app/routes/app.delivery-customization.$functionId.$id.tsx

```diff
@@ -0,0 +1,238 @@
+import { useState, useEffect } from "react";
```

Comment:
Brand new file, because it looks like this was accidentally deleted from the sample app the last time this was updated.